azure-pipelines.yml

```diff
@@ -0,0 +1,321 @@
+variables:
+- group: sonarsource-build-variables
+- name: BuildParameters.solution
+  value: SonarLint.VisualStudio.Integration.sln
+trigger:
+  branches:
+    include:
+    - master
+    - dev/*
+    - feature/*
+    - branch-*
+  batch: True
+jobs:
+- job: Phase_1
+  strategy:
+    maxParallel: 2
+    matrix:
```

Comment:
running multiple configuration from a variable is not supported in yaml so I had to change it to a matrix strategy.